egs2/voxtlm/lm1/local/prepare_lm_data.py

```diff
@@ -0,0 +1,198 @@
+import argparse
+import random
+import shutil
+from pathlib import Path
+from typing import Union
+
+
+def read_text(text: Path):
+    uttid2text = {}
+    with text.open("r") as fp:
+        for line in fp.readlines():
+            line = line.strip().split()
+            uttid2text[line[0]] = " ".join(line[1:])
+    return uttid2text
+
+
+# Convert discrete unit to CJK characters
+# CJK Unified Ideographs (20,977 chars): \u4e00 - \u9fff
```

Comment:
Quick question here on whether the usage of CJK token would be good: my understanding for CJK is its easiness of compute BPE, however we have following constraints/issues
- cluster size (limits to a boundary)
- worse readability (as the CJK mapping is not quite obvious)
- meaningless for multi-stream modeling of speech: as for multi-stream (e.g., neural codec etc) we cannot achieve bpe on each stream, thus cannot gain from the benefits of achieving BPE easily.

Please let me know if you have other thoughts~ I guess the decision here would be very fundamental as it will also affects the modeling in many other tasks:
- in s2st and svs2, we directly use text_int (sequence of integer)
- in asr2 and st2, we use cjk to reduce sequence
- we are still in discussion of which format to use in tts2

Reply:
For another comment on the design: whether we would want the cjk conversion in local or within a certain stage?

Reply:
Since we do extraction of data on stage 2 and stage 3, we have to convert cjk after or within stage 3. But I agree we can do that outside local as well.

Reply:
Maybe we can add the CJK token mapping with BPE, as it is needed for BPE.

Reply:
Added CJK token as a parameter, can be turned off depending on recipe.